SPECS/rrd-client-lib.spec

```diff
@@ -0,0 +1,60 @@
+%define debug_package %{nil}
+
+Name:           rrd-client-lib
+Version:        1.0.0
+Release:        1%{?dist}
+Summary:        C library for writing RRDD plugins
+License:        MIT
+Group:          Development/Other
+URL:            https://github.com/xapi-project/rrd-client-lib/
+Source0:        https://github.com/xapi-project/rrd-client-lib/archive/v%{version}/rrd-client-lib-%{version}.tar.gz
+BuildRequires:  zlib
+BuildRequires:  gcc
+BuildRequires:  ocaml-rrd-transport-devel
+
+%description
+Library for writing RRDD plugins in C. This package contains just the
+dynamic library but no header files or static libraries.
+
+%package devel
+Group:          Development/Libraries
+Summary:        Libraries and header files for rrd development
+Requires:       rrd-client-lib = %{version}-%{release}
+
+%description devel
+C library for writing RRDD plugins - including header files
+
+%prep
+%setup -q
+
+%build
+make 
+
+%install
+rm -rf %{buildroot}
+install -d %{buildroot}%{_libdir}
+install -d %{buildroot}%{_includedir}
+
+install librrd.so  %{buildroot}%{_libdir}
+install librrd.h   %{buildroot}%{_includedir}
+install librrd.a   %{buildroot}%{_libdir}
+
+%post   -p /sbin/ldconfig
+%postun -p /sbin/ldconfig
+
+%clean
+rm -rf %{buildroot}
+
+%files devel
+%defattr(-,root,root)
+%{_includedir}/librrd.h
+%{_libdir}/librrd.a
+
+%files
```

Comment:
It's conventional to put the main package's `%files` stanza above the subpackage's stanza.